helpers/__tests__/init-test.ts

```diff
@@ -34,7 +35,7 @@ beforeEach(() => {
 });
 
 afterAll(() => {
-  Initializer.setLocale('en').setMessages({}).setUrlContext('');
+  Initializer.setLocale('en').setMessages({}).setUrlContext('').setReactDomContainer(undefined);
```

Comment:
TypeScript isn't angry here because the function definition is `setReactDomContainer(nodeSelector: string)` ?

Reply:
Just checked out the branch. TypeScript is angry ^^.

Reply:
Heee... That's really weird that the validation passes indeed...

Reply:
And why is it not angry on my side ???

Reply:
That's really weird... On my machine it's angry if I try to put in a number or object but not for undefined or null...

Reply:
The `ts-check` task runs successfully on my machine, but I have the TypeScript error in my editor. 🤯 
IntelliJ uses the TypeScript version from sonar-ui-common, 3.8.3.

Reply:
Got the same version too :exploding_head: